app/experimenter/experiments/tests/test_forms.py

```diff
@@ -202,6 +205,302 @@ class Meta:
+            "survey_instructions": "some blurb about instructions",
+            "slug": "the-experiment-slug",
+            "normandy_id": 4321,
+            "data_science_bugzilla_url": "https://bugzilla.mozilla.org/123/",
+            "feature_bugzilla_url": "https://bugzilla.mozilla.org/124/",
+            "risk_internal_only": True,
+            "risk_partner_related": True,
+            "risk_brand": True,
+            "risk_fast_shipped": True,
+            "risk_confidential": True,
+            "risk_release_population": True,
+            "risk_revenue": True,
+            "risk_data_category": True,
+            "risk_external_team_impact": True,
+            "risk_telemetry_data": True,
+            "risk_ux": True,
+            "risk_security": True,
+            "risk_revision": True,
+            "risk_technical": True,
+            "risk_technical_description": "tech description",
+            "testing": "some blurb about testing",
+            "test_builds": "some blurb about test builds",
+            "qa_status": "some blurb qa status",
+            "review_science": True,
+            "review_engineering": True,
+            "review_qa_requested": True,
+            "review_intent_to_ship": True,
+            "review_bugzilla": True,
+            "review_qa": True,
+            "review_relman": True,
+            "review_advisory": True,
+            "review_legal": True,
+            "review_ux": True,
+            "review_security": True,
+            "review_vp": True,
+            "review_data_steward": True,
+            "review_comms": True,
+            "review_impacted_teams": True,
+        }
+
+        class TestForm(ChangeLogMixin, forms.ModelForm):
+
+            class Meta:
+                model = Experiment
+                fields = "__all__"
+
+        class TinyTestForm(ChangeLogMixin, forms.ModelForm):
+
+            class Meta:
+                model = Experiment
+                fields = ("review_impacted_teams",)
+
+        form = TestForm(request=self.request, data=data, instance=experiment)
+
+        self.assertTrue(form.is_valid())
+        experiment = form.save()
+
+        self.assertEqual(experiment.changes.count(), 1)
+
+        self.assertEqual(experiment.changes.latest().new_values, data)
+
+        self.assertTrue(experiment.review_impacted_teams)
+        form = TinyTestForm(
+            request=self.request,
+            data={"review_impacted_teams": False},
+            instance=experiment,
+        )
+
+        self.assertTrue(form.is_valid())
+        form.save()
+        self.assertEqual(experiment.changes.count(), 2)
+        self.assertEqual(
+            experiment.changes.latest().old_values,
+            {"review_impacted_teams": True},
+        )
+        self.assertEqual(
+            experiment.changes.latest().new_values,
+            {"review_impacted_teams": False},
+        )
+
+    def test_mixin_sets_old_values_and_new_values_with_m2m(self):
```

Comment:
The name suggests this is testing old and new values for m2m fields, but down in the assertions it looks like it's only testing new values for the m2m fields, doesn't seem to match the name?